luxonis_ml/data/requirements.txt

```diff
@@ -10,3 +10,4 @@ filelock~=3.0
 bidict~=0.21
 gdown~=4.7
 defusedxml~=0.7
+pillow-heif<0.22.0
```

Comment:
New version of `pillow-heif` released on March 15th cannot be installed on macOS